app/scripts/views/sign_up.js

```diff
@@ -192,7 +193,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, Session, AuthErrors,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      console.log(this._able);
+      console.log(this._able.choose('mailcheckEnabled'));
```

Comment:
> this is always returning false

Yes, `startDate` is required to enable the experiment.



> Is it something that can be added to able or should I work on adding the UUID?

I haven't added a `sessionId` yet. I haven't thought of how to implement it yet, probably a cookie.